scripts/updateManyTasksWithratingAllowed.js

```diff
@@ -0,0 +1,19 @@
+require('../config/index');
+const TasksQuery = require('../models/tasks')
+
+let updateTasks = async () => {
+
+    let payload = {ratingAllowed:{$exists:false}}
+    let insertRatingAllowedKey = {$set:{ratingAllowed:true}}
+
+    let ratingAllowedRes =  await TasksQuery.updateMany(payload,insertRatingAllowedKey,(err, resp) => {
+        if (err){
+            console.log("error updating user",err.message);
```

Comment:
in updateMany i think there is no message key on error object

Reply:
removed and changed to err only